### PR TITLE
Chore/remove atheris test payload tools

### PR DESCRIPTION
**Pull request by Karrenbelt**
## Proposed changes

I chose the most reasonable option and simply deleted the remaining atheris fuzzer tests in test_payload_tools.py. 
- No changes in coverage gaps
- will remove `atheris` from tox.ini in base PR #1613, which I need to merge main into anyway

## Fixes

If it fixes a bug or resolves a feature request, be sure to link to that issue.

## Types of changes

What types of changes does your code introduce? (A **breaking change** is a fix or feature that would cause existing functionality and APIs to not work as expected.)
_Put an `x` in the box that applies_

- [ ] Non-breaking fix (non-breaking change which fixes an issue)
- [ ] Breaking fix (breaking change which fixes an issue)
- [ ] Non-breaking feature (non-breaking change which adds functionality)
- [ ] Breaking feature (breaking change which adds functionality)
- [ ] Refactor (non-breaking change which changes implementation)
- [ ] Messy (mixture of the above - requires explanation!)

## Checklist

_Put an `x` in the boxes that apply._

- [ ] I have read the [CONTRIBUTING](../blob/main/CONTRIBUTING.md) doc
- [ ] I am making a pull request against the `main` branch (left side). Also you should start your branch off our `main`.
- [ ] Lint and unit tests pass locally with my changes
- [ ] I have added tests that prove my fix is effective or that my feature works
- [ ] I have locally run services that could be impacted and they do not present failures derived from my changes

## Further comments

If this is a relatively large or complex change, kick off the discussion by explaining why you chose the solution you did and what alternatives you considered, etc...

### CODE REVIEW

File: packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py [hunk 1]
```diff
@@ -101,105 +92,3 @@ def test_payload_to_hex_and_back() -> None:
...
-            return
-        verification = VerificationStatus(verification_int)
-        tx_hash = fdp.ConsumeString(12)
-        if len(tx_hash) != 64:
-            return
-        tx_hist_payload_to_hex(verification, tx_hash)
-
-    atheris.instrument_all()
-    atheris.Setup(sys.argv, fuzz_tx_hist_payload_to_hex)
-    atheris.Fuzz()
```
Comment (Karrenbelt) [inline packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py:125]: This piece of code does not warrant the use of a fuzzer
https://github.com/valory-xyz/open-autonomy/blob/faabe6263607775720aad863211d0ca6fd17d937/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L58-L70

File: packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py [hunk 2]
```diff
@@ -101,105 +92,3 @@ def test_payload_to_hex_and_back() -> None:
...
-        fdp = atheris.FuzzedDataProvider(input_bytes)
-        try:
-            payload = fdp.ConsumeString(12)
-            tx_hist_hex_to_payload(payload)
-        except PayloadDeserializationError:
-            pass
-
-    atheris.instrument_all()
-    atheris.Setup(sys.argv, fuzz_tx_hist_hex_to_payload)
-    atheris.Fuzz()
```
Comment (Karrenbelt) [inline packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py:144]: This piece of code does not warrant the use of a fuzzer
https://github.com/valory-xyz/open-autonomy/blob/faabe6263607775720aad863211d0ca6fd17d937/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L73-L88

File: packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py [hunk 3]
```diff
@@ -101,105 +92,3 @@ def test_payload_to_hex_and_back() -> None:
...
-            to_address,
-            data,
-            operation,
-            base_gas,
-            safe_gas_price,
-        )
-
-    atheris.instrument_all()
-    atheris.Setup(sys.argv, fuzz_hash_payload_to_hex)
-    atheris.Fuzz()
```
Comment (Karrenbelt) [inline packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py:184]: This piece of code does not warrant the use of a fuzzer
https://github.com/valory-xyz/open-autonomy/blob/faabe6263607775720aad863211d0ca6fd17d937/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L91-L143

File: packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py [hunk 4]
```diff
@@ -101,105 +92,3 @@ def test_payload_to_hex_and_back() -> None:
...
-        if len(payload) < 500:
-            return
-        try:
-            skill_input_hex_to_payload(payload)
-        except TypeError:
-            pass
-
-    atheris.instrument_all()
-    atheris.Setup(sys.argv, fuzz_skill_input_hex_to_payload)
-    atheris.Fuzz()
```
Comment (Karrenbelt) [inline packages/valory/skills/transaction_settlement_abci/tests/test_payload_tools.py:205]: This piece of code does not warrant the use of a fuzzer
https://github.com/valory-xyz/open-autonomy/blob/faabe6263607775720aad863211d0ca6fd17d937/packages/valory/skills/transaction_settlement_abci/payload_tools.py#L146-L162

**Comment by Karrenbelt** [comment]
failures:
- windows 3.10 - I will address this in base PR as it is related to it, and not this PR
    ```
    2022-12-05T22:49:49.2527421Z     @given(st.binary())
    2022-12-05T22:49:49.2528262Z >   def test_fuzz_send_signing_request(self, input_bytes: bytes) -> None:
    ...
    2022-12-05T22:49:49.2534869Z E           hypothesis.errors.Flaky: Hypothesis test_fuzz_send_signing_request(self=<packages.valory.skills.abstract_round_abci.tests.test_behaviours_utils.TestBaseBehaviour at 0x1f5cbb474c0>, input_bytes=b'') produces unreliable results: Falsified on the first call but did not on a subsequent one
    ...
    2022-12-05T22:51:36.9479084Z Exit Code	Package
    2022-12-05T22:51:36.9479525Z 1       	valory\skills\abstract_round_abci
    ```

**Review by 0xArdi — APPROVED** [review]

**Review by DavidMinarsch — APPROVED** [review]

**Pull request merged**